libraries/AP_Logger/AP_Logger_DataFlash.cpp

```diff
@@ -254,18 +250,38 @@ void AP_Logger_DataFlash::SectorErase(uint32_t sectorNum)
     WITH_SEMAPHORE(dev_sem);
 
     uint32_t PageAdr = sectorNum * df_PageSize * df_PagePerSector;
+    // CS required according to https://www.winbond.com/resource-files/w25q128fv%20rev.l%2008242015.pdf
+    dev->set_chip_select(true);
```

Comment:
the dev->transfer() should already be setting the CS automatically. We only need set_chip_select() if we need the CS held over multiple operations

Reply:
ok ... but why then does AP_Logger_DataFlash::PageToBuffer have it?

Reply:
because it does two transfers, one to set the address, then one to transfer. These need to have CS set around the whole transaction

Reply:
ok done